ntbs-service/Pages/Alerts/TransferRequestAction.cshtml.cs

```diff
@@ -151,7 +151,8 @@ public async Task AcceptTransferAndDismissAlertAsync()
             Service.UpdateHospitalDetailsWithoutSave(Notification, Notification.HospitalDetails);
             _treatmentEventRepository.AddWithoutSave(transferOutEvent);
             _treatmentEventRepository.AddWithoutSave(transferInEvent);
-            await _alertService.DismissAlertAsync(TransferAlert.AlertId, User.Username());
+            await _alertService.DismissAlertWithoutSaveAsync(TransferAlert.AlertId, User.Username());
+            await _treatmentEventRepository.SaveChangesAsync(NotificationAuditType.Edited);

```

Comment:
Something that I should have thought of first time around: this code makes it really clear that we want to save all these changes in one batch, but it may still be confusing to future developers why this is the case. I think we should put a comment above these 5 lines giving a brief description of the bug that we're fixing, how we're fixing it, and a Jira reference.

Reply:
Added 👍